src/utils/saml.ts

```diff
@@ -670,18 +672,103 @@ export const getPreValidateResponse = (
     return callback(null, true, _.InResponseTo);
   };
 
+  const extractAndLogTimings = (
```

Comment:
Add a comment for this function. What it does, how it does.

Reply:
This functions should works with any SPID/CIE saml, can we move the function declaration in `samlUtils`?

Reply:
sure